crates/ruff_python_formatter/src/other/except_handler_except_handler.rs

```diff
@@ -25,7 +39,7 @@ impl FormatNodeRule<ExceptHandlerExceptHandler> for FormatExceptHandlerExceptHan
         let comments_info = f.context().comments().clone();
         let dangling_comments = comments_info.dangling_comments(item.as_any_node_ref());
 
-        write!(f, [text("except")])?;
+        write!(f, [text("except"), self.has_star.then(|| text("*"))])?;
```

Comment:
Nit
```suggestion
        write!(f, [text("except"), self.has_star.then_some(text("*"))])?;
```

Calling `text` is cheap (it compiles to a const)

Reply:
wasn't saving the call, just missed that method existing 😆